participant-manager-datastore/participant-manager-service/src/main/java/com/google/cloud/healthcare/fdamystudies/mapper/SiteMapper.java

```diff
@@ -22,6 +22,7 @@ private SiteMapper() {}
   public static SiteResponse toSiteResponse(SiteEntity site) {
     SiteResponse response = new SiteResponse();
     response.setSiteId(site.getId());
+    response.setSiteName(site.getLocation().getName());
```

Comment:
Please update file copyright to `2020-2021`

Reply:
Updated file to `2020-2021`.